packages/components/date-picker/src/date-picker-com/basic-date-table.vue

```diff
@@ -68,6 +70,8 @@ const lastRow = ref<number>()
 const lastColumn = ref<number>()
 const tableRows = ref<DateCell[][]>([[], [], [], [], [], []])
 
+const focusWithClick = ref(false)
```

Comment:
Maybe there is no need to use `ref` here

```suggestion
let focusWithClick = false
```

Reply:
you're right. 👍